client/app/services/file-manager.js

```diff
@@ -21,6 +21,8 @@ export default class FileManager {
     this.filesToUpload = filesToUpload; // EmberFileUpload QUEUE Object
   }
 
+  @tracked existingFiles;
```

Comment:
Makes sense, to make sure templates/components are reacting to changes in existingFiles (if I'm understanding this change correctly).